course_catalog/etl/enrollments.py

```diff
@@ -0,0 +1,78 @@
+"""Course run enrollments"""
+import pyathena
+import pytz
+from pyathena.cursor import DictCursor
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from course_catalog.models import LearningResourceRun, Course, Enrollment
+
+
+def athena_cursor():
+    """Returns authorized pyathena cursor"""
+    return pyathena.connect(
+        aws_access_key_id=settings.ATHENA_ACCESS_KEY_ID,
+        aws_secret_access_key=settings.ATHENA_SECRET_ACCESS_KEY,
+        work_group=settings.ATHENA_WORK_GROUP,
+        cursor_class=DictCursor,
+        region_name=settings.ATHENA_REGION_NAME,
+    ).cursor()
+
+
+def update_enrollments_for_user(user):
+    """
+    Updates enrollment data for a singe user
+
+    Args:
+        user (User): user to fetch enrollments for
+    """
+
+    if not (
+        settings.ATHENA_ACCESS_KEY_ID
+        and settings.ATHENA_SECRET_ACCESS_KEY
+        and settings.ATHENA_WORK_GROUP
+        and settings.ATHENA_REGION_NAME
+    ):
```

Comment:
It seems like this function shouldn't have any awareness of the settings requirements for the cursor object and should be external to the logic to process the query results.

Reply:
I was using the credentials as a feature flag so we don't get noisy errors if this is merged before the credentials are set. If we are going to just use the existing AWS credentials, this isn't necessary. The only one of these credentials that we will keep is ATHENA_WORK_GROUP, and 'primary' will be the default so it will work without any credentials set